controller/src/controller/models/domain/rsptag.py

```diff
@@ -122,8 +107,8 @@ class RSPImageTag:
     image_type: RSPImageType
     """Type (release series) of image identified by this tag."""
 
-    version: Version | None
-    """Version information as a semantic version."""
+    version: str | None
+    """Version information as a string representing a semantic version."""
```

Comment:
No, I think this change is wrong. This should stay a `Version`. It shouldn't be necessary to serialize this object. That will remove the need for a bunch of the changes below.

Reply:
I'm pretty sure I got into this mess in the first place by having it be a `Version` internally.  I'll give it a try, but I think it's going to fail.

Reply:
Oh, maybe I see, leave it a str in the policy (and do the parse()) but a Version on the image/tag